src/main/kotlin/com/exactpro/th2/ws/client/api/impl/DefaultHandler.kt

```diff
@@ -34,6 +36,10 @@ class DefaultHandler : IHandler {
         }
     }
 
+    override fun preOpen(clientSettings: IClientSettings) {
+        settings.defaultHeaders.forEach { (header, values) -> clientSettings.addHeaders(header, values) }
```

Comment:
```suggestion
        settings.defaultHeaders.forEach(clientSettings::addHeaders)
```

Reply:
This one will work only if we move the `addHeaders` method to the interface. If we leave it as extension method the function signature won't match the required one

Reply:
I've tried it in IDE (didn't run the code though) and it has no problems in resolving a proper method

Reply:
Hm...yes, you are right. I have tried it again and everything compiles. Probably, I did something wrong the first time